antenna-documentation/src/site/markdown/sw360-data-model.md.vm

```diff
@@ -24,13 +24,14 @@ In the following section there is an outline what data of SW360 ${docNameCap} ma
   | External Ids | Additional information can be put here in a `Map<String,String>` format |
 
 #[[###]]# Licenses
- All licenses are created separately from the components and releases.
- Enriching and updating is also done separately.
+ All licenses are stored separately from the components and releases.
+ Enriching is also done as part of the enriching process. Updating is not done actively, since
```

Comment:
This sentence is not completely clear, in my opinion.
1. Why "also"? 
2. I would write "Enriching of license information" instead of just enriching, to make it more clear.